### PR TITLE
Building pubsite with docker

### PR DESCRIPTION
**Pull request by devinmatte**
We need a more modern build system for the pubsite, this now uses docker multistage builds

### CODE REVIEW

File: .dockerignore
Comment (mxmeinhold) [inline .dockerignore:3]: Is the version info in the footer hardcoded rather than determined from git context?

**Review by mxmeinhold — APPROVED** [review]

File: .dockerignore
Comment (devinmatte) [reply .dockerignore:3]: Do we have a version on the footer of the public site?

Comment (RamZallan) [reply .dockerignore:3]: image
The footer just has a link to the GH repo itself, no like commit hash or anything

Comment (devinmatte) [reply .dockerignore:3]: That's what I thought. We should be good to ignore it.

**Comment by devinmatte** [comment]
I'm going to merge into `develop`. Will need a new PR in a few minutes for `master`

**Review by RamZallan — APPROVED** [review]

**Comment by RamZallan** [comment]
lol approved after you merged woops

**Pull request merged**
